bootstrap/Old_BootStrapBackup/_bootstrap_common.py

```
@@ -0,0 +1,340 @@
+    for env_file in env_candidates:
+        env_map = load_dotenv(env_file)
+        if env_map.get("DATABASE_URL"):
+            return env_map["DATABASE_URL"]
+
+    return "postgresql+psycopg2://postgres:root@localhost:5432/farmos"
+
+
+def resolve_command(command: list[str]) -> list[str]:
+    """Windows의 cmd/bat 실행을 안전하게 보정한다."""
+    if not command:
+        raise BootstrapError("빈 명령은 허용되지 않습니다.")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)